src/metaschema/oscal_implementation-common_metaschema.xml

```diff
@@ -385,8 +388,8 @@
                   </assembly>
                   <field ref="role-id" min-occurs="0" max-occurs="unbounded">
                         <group-as name="role-ids" in-json="ARRAY"/>
-                  </field>
-                  <assembly ref="authorized-privilege" max-occurs="unbounded">
+                  </field>                  
+                  <assembly ref="authorized-privilege" max-occurs="unbounded" deprecated="1.1.3">
```

Comment:
@david-waltermire should this be `deprecated="1.2.0"` to signal that support for user-based `authorized-privilege`?